canonical-web-and-design/react-components
# Initial implementation of CodeSnippet component

## Done

- React implementation of new Vanilla Code snippet component

Fixes #373

## QA

### QA steps

- Open demo: https://react-components-381.demos.haus/?path=/docs/codesnippet--default-story
- Make sure all the examples work as expected
- Make sure documentation and examples are clear on how to use the component


### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```
